From 63ac6ba91254b7cf5a3fd4376ec0124f87d1a53e Mon Sep 17 00:00:00 2001
From: Pier Luigi Fiorini <pierluigi.fiorini@gmail.com>
Date: Sun, 7 Oct 2012 14:23:45 +0200
Subject: [PATCH][BUILD] Remove deprecated dist-lzma.

The dist-lzma, used to produce .tar.lzma archives has been removed.
For more information see:
  http://www.flameeyes.eu/autotools-mythbuster/forwardporting/automake.html

Signed-off-by: Pier Luigi Fiorini <pierluigi.fiorini@gmail.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 31f0c7e..04b7cf9 100644
--- a/configure.ac
+++ b/configure.ac
@@ -13,7 +13,7 @@ AC_INIT([GNU Texinfo], [4.13], [bug-texinfo@gnu.org])
 
 dnl Must come before AM_INIT_AUTOMAKE.
 AC_CONFIG_AUX_DIR([build-aux])
-AM_INIT_AUTOMAKE([1.10.1 readme-alpha dist-lzma])
+AM_INIT_AUTOMAKE([1.10.1 readme-alpha])
 
 # Minimum Autoconf version required.
 AC_PREREQ(2.59)
-- 
1.7.12.2

